index.html

```diff
@@ -0,0 +1,206 @@
+<html>
+    <head>
+        <title>YouTube</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body> 
+
+<nav class="flex-div">
+    <div class="nav-left flex-div">    
+    <img src="images/menu.png" class="menu-icon">
+    <img src="images/logo.png" class="logo">
+        </div>
+        <div class="nav-middle flex-div">
+            <div class="search-box flex-div">
+
+                <input type="search" placeholder="search">
+                <img src="images/search.png" class="search-icon">
+            </div>
+            <img src="images/voice-search.png" class="mic-icon">
+        </div>
+        <div class="nav-right flex-div">
+            <img src="images/upload.png">
+            <img src="images/more.png">
+            <img src="images/notification.png">
+            <img src="images/Jack.png" class="user-icon">
+            
+        </div>
+    </nav>
+
+
+    <!-- -----------side-bar------------- -->
+
+    <div class="side-bar">
```

Comment:
- [ ] [OPTIONAL] This comment is nearly perfect, it's polite, clear and descriptive, however, I believe it could be further improved by referring to the official project requirements either by using a link to the[ requirements page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) or by quoting the requirement specific to this case like this 🔽 
> Use the appropriate tags for each element (e.g., links, titles, etc.) and use HTML5 semantic tags (e.g., header, nav) over divs.